frontend/src/components/forms/CreateLog.tsx

```diff
@@ -68,12 +125,17 @@ const CreateLog = () => {
     }),
   );
   const [building, setBuilding] = useState("");
-  const [resident, setResident] = useState("");
+  const [resident, setResident] = useState(-1);
```

Comment:
why use -1 instead of an empty state?

Reply:
I wanted to store the value of each field rather than their string representation (since the string representation isn't very useful in context of the API)